index.html

```diff
@@ -69,5 +69,181 @@ <h1>Hey There. I’m Ronald.</h1>
         </li>
       </ul>
     </main>
+    <!-- works section -->
+    <section class="works">
+      <h2 class="recent">My Recent Works</h2>
+
+      <div class="card">
+        <div class="projimage">
+          <img src="./images/projects/Img Placeholder.svg" alt="" />
+        </div>
+        <h2 class="cardheader">Multi-Post Stories</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="languages">
+          <li>CSS</li>
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button>See project</button>
+      </div>
+      <div class="card2">
+        <div class="textitems">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button>See project</button>
+      </div>
+      <div class="card2">
+        <div class="textitems">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button>See project</button>
+      </div>
```

Comment:
- [ ] For the class `.card2`, the image shows how the project is currently. You have a `color: #fff` on the `.card2` class which is making the text invisible due to the already white background. Please fix this.

![image](https://github.com/mj-e-boc/M-J-Portfolio/assets/84156714/d96fb701-7761-4bc8-8d31-b20766b03d9d)